forest/daemon/src/daemon.rs

```diff
@@ -361,11 +363,6 @@ pub(super) async fn start(config: Config, detached: bool) -> Db {
 
     select! {
         () = sync_from_snapshot(&config, &state_manager).fuse() => {},
-        _ = ctrlc_oneshot => {
-            // Cancel all async services
-            services.shutdown().await;
-            return db;
-        },
```

Comment:
You can't just delete this code. It was here for a reason.

Reply:
The code is required for a graceful shutdown. We _do_ want a graceful shutdown. That's the entire point of the ctrl-c handler.

Reply:
We are still doing a graceful shutdown using the same commands in the end of the `start` function

Reply:
If ctrl-c is hit duing `sync_from_snapshot`, how do we do a graceful shutdown?

Reply:
Keep in mind that `sync_from_snapshot` may take 10 minutes to complete.

Reply:
Reverted